editor/src/clj/editor/keymap.clj

```diff
@@ -81,7 +81,7 @@
            ["Meta+D" :select-next-occurrence]
            ["Meta+Delete" :delete-to-end-of-line]
            ["Meta+Down" :end-of-file]
-           ["Meta+E" :hide-selected]
+           ["Meta+E" :toggle-selected]
```

Comment:
`:toggle-selected` kind of sounds like it is toggling whether or not an object is selected. Perhaps we could name it something like `:hide-toggle-selected`?

I think it would fit in well with the related Show and Hide actions:
```
:hide-toggle-selected
:hide-unselected
:show-last-hidden
:show-all-hidden
```

Reply:
I fixed in the last commit.